test/elixir/test/reshard_basic_test.exs

```diff
@@ -109,7 +109,7 @@ defmodule ReshardBasicTest do
     resp = Couch.get("/#{db}/_shards")
     assert resp.status_code == 200
     shards = resp.body["shards"]
-    assert node1 not in shards["00000000-ffffffff"]
+    # assert node1 not in shards["00000000-ffffffff"]
```

Comment:
@nickva I've had to comment out this line. Does this still make the test valid?

Reply:
Let's replace it with

```
assert node1 not in Map.get(shards, "00000000-ffffffff", [])
```

This way it will still check that the original range was gone and work on n=1 and n>2 cases

(Added a bit of extra explanation in a similar assert below)